### PR TITLE
Fix deadlocked builds

### PR DESCRIPTION
**Pull request by ptomato**
https://phabricator.endlessm.com/T14477

### CODE REVIEW

File: tests/js/app/interfaces/testCard.js [hunk 1]
```diff
@@ -141,9 +141,8 @@ describe('Card interface', function () {
 
     it('sets a thumbnail frame visible if model has a thumbnail uri', function () {
         let engine = MockEngine.mock_default();
-        engine._lookup_ekn_uri.and.callFake(() => {
-            return Gio.Vfs.get_local().get_file_for_path('/dev/zero');
-        });
+        let zeroes = Gio.Vfs.get_local().get_file_for_path('/dev/zero');
```
Comment (mattdangerw) [inline tests/js/app/interfaces/testCard.js]: I wonder if we even need to single out vfs local for this. Shouldn't `Gio.File.new_for_uri('file:///dev/zero')` work?

Comment (mattdangerw) [inline tests/js/app/interfaces/testCard.js]: Do we even need to single out Gio.Vfs.get_local() here? Wouldn't `let zeroes = Gio.File.new_for_uri('file:///dev/zero');` work?

Maybe I'm not understanding the vfs setup completely...

File: configure.ac [hunk 2]
```diff
@@ -223,8 +223,11 @@ EOS_CHECK_GJS_GIR([GLib], [2.0])
 EOS_CHECK_GJS_GIR([GObject], [2.0])
 EOS_CHECK_GJS_GIR([Gtk], [3.0])
 EOS_CHECK_GJS_GIR([Pango], [1.0])
+EOS_CHECK_GJS_GIR([Rsvg], [2.0])
```
Comment (magcius) [inline configure.ac]: unrelated???

**Comment by mattdangerw** [comment]
CR pass

File: tests/js/app/interfaces/testCard.js (hunk 1 above)
Comment (dbnicholson) [inline tests/js/app/interfaces/testCard.js]: Or even just `Gio.File.new_for_path('/dev/zero')` should be fine here, right?

Comment (mattdangerw) [inline tests/js/app/interfaces/testCard.js]: Yar true. But just merged

Comment (ptomato) [inline tests/js/app/interfaces/testCard.js]: `new_for_path` was what was actually merged
